src/Pages/Dictionary.js

```diff
@@ -1,10 +1,38 @@
-import React from 'react';
-import WordTermDisplay from '../Components/WordTermDisplay';
+import React, { useState, useEffect } from 'react';
+import WordTerm from '../Components/WordTerm';
 
-function Dictionary() {
-  return (
-    <WordTermDisplay />
-  );
+// airtable configuration
+const Airtable = require('airtable');
+
+const airtableConfig = {
+  apiKey: process.env.REACT_APP_AIRTABLE_USER_KEY,
+  baseKey: process.env.REACT_APP_AIRTABLE_BASE_KEY,
+};
+
+const base = new Airtable({ apiKey: airtableConfig.apiKey })
+  .base(airtableConfig.baseKey);
+
+function WordTermDisplay() {
+  const [definitions, setDefinitions] = useState([]);
+
+  const getWordInfo = () => {
+    base('Definition').select({ view: 'Grid view' }).all()
+      .then((records) => {
```

Comment:
It would be good to add error checking here! All airtable calls return an error object along with the records to show if the call failed in anyway. If you look at some of your past work you can see examples of this, but if you want the exact syntax ask me!